v2/components/checkbox/checkbox.tsx

```diff
@@ -28,37 +28,3 @@ export const Checkbox = (props: {value?: boolean; onChange?: (value: boolean) =>
         </div>
     );
 };
-
-export interface CheckboxOption {
-    label: string;
-    count?: number;
-    icon?: React.ReactNode;
-}
-
-export const CheckboxRow = (props: {value: boolean; onChange?: (value: boolean) => void; option: CheckboxOption}) => {
-    const [value, setValue] = React.useState(props.value);
-
-    React.useEffect(() => {
-        setValue(props.value);
-    }, [props.value]);
-
-    return (
-        <div className={`checkbox__item ${value ? 'checkbox__item--selected' : ''}`} onClick={() => setValue(!value)}>
```

Comment:
as I mentioned in https://github.com/argoproj/argo-cd/pull/6990/files#r688692844 please move related CSS styles as well